src/main/resources/view/EventWindowCard.fxml

```diff
@@ -22,7 +22,9 @@
 
       <Label fx:id="eventName" text="\$eventName" styleClass="cell_big_label" wrapText="true"/>
       <Separator />
-      <Label fx:id="dateAndTime" styleClass="cell_small_label" text="\$dateAndTime" />
+      <Label fx:id="startDate" styleClass="cell_small_label" text="\$startDate" />
+      <Separator />
```

Comment:
small suggestion, maybe start date and end date no need separate since they are kinda part of the same category

Reply:
hmm how shd i do it

(start date) - (end date)
(start date) to (end date)
from (start date) to (end date)

Reply:
UI expert pls help 😵‍💫

Reply:
oh wait i meant remove the `<Separator>` HAHAHAH

Reply:
<img width="612" alt="Screenshot 2023-10-23 at 9 15 32 PM" src="https://github.com/AY2324S1-CS2103T-T14-4/tp/assets/15770059/a2e78e78-3d1f-47c2-bc8f-583a46587a2f">

like this ahh

Reply:
also should i help increase the height of this window such that you can see every event detail LOL